constitution.tex

```diff
@@ -275,18 +275,23 @@ \subsection{Modifying the Constitution}
 	\item Receive priority for on-floor housing, or apply for On-Floor Status
 \end{enumerate}
 
-\asubsubsection{Expectations of Voting Members}
+\asubsubsection{Qualifications of Voting Members}
 \renewcommand{\theenumi}{\arabic{enumi}} % For this section, we want items to use letters
 
-The following requirements must be met by the beginning of the Intro Eval for an Active Member to be allowed to vote.
-Any of these requirements may be waived by the Evals Director or an E-Board Vote.
+For an Active Member to be allowed to vote in an Intro Eval, they must meet the following requirements by the beginning of the Intro Eval.
+Any of these requirements may be waived by the Evals Director or an E-Board Vote. An Active Member who meets these requirements may opt out of Voting Membership by submitting a request to the Evals Director or E-Board.
```

Comment:
Formatting - style recommendation to place individual sentences on separate lines.

Reply:
I don't know what you want specifically. Currently, 4.B.4.A is consistent with 4.B.3.A and 4.B.5.A. If you suggest changes, I'll review them (or just add a commit).

Reply:
Line 282 has two sentences on it; the style guidelines would recommend one sentence per line.